io.catenax.manufactured_parts_quality_information/RELEASE_NOTES.md

```diff
@@ -2,6 +2,16 @@
 All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
+- Version 2.0.0 of model is deprecated. It has a wrong regex for BPNS validation from the old shared model for BPNS trait.
+
+## [2.1.0] - 2024-02-12
```

Comment:
update the date

Reply:
I'm not able to do that due to missing rights on this branch